### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by DonchenkoIgor** [Laravel-Backpack/community-forum#1002] [Q&A (Help)]
Install the Tabler theme ............................................................................................... DONE
.. ERROR                                                                                                                       
  │ For further information please check the log file.                                                                         
  │ You can also follow the manual installation process documented on Github.

**Comment by jcastroa87** [comment]
Hello @DonchenkoIgor 

Please check you log file if there is some information about error. You can follow the manual installation as our [docs](https://github.com/Laravel-Backpack/theme-tabler?tab=readme-ov-file#manual-installation) show.

Let me know if there is some details about this.

Cheers.

**Reply by DonchenkoIgor** [reply]
[2024-06-02 14:28:12] local.ERROR: '""' is not recognized as an internal or external command,
operable program or batch file.

**Comment by DonchenkoIgor** [comment]
When executing a command: 
->  php artisan backpack:require:theme-tabler

I get this error:
  Install the Tabler theme ............................................................................................... DONE
.. ERROR
  │ For further information please check the log file.
  │ You can also follow the manual installation process documented on Github.

please help me solve the error

**Reply by jcastroa87** [reply]
Please try the manual install:

```
composer require backpack/theme-tabler
```

Let me know the versions you are using too
```
php artisan backpack:version
```

Cheers.

**Reply by DonchenkoIgor** [reply]
### PHP VERSION:
8.1.9

### PHP EXTENSIONS:
Core, bcmath, calendar, ctype, date, filter, hash, iconv, json, SPL, pcre, readline, Reflection, session, standard, mysqlnd, tokenizer, zip, zlib, libxml, dom, PDO, bz2, SimpleXML, xml, xmlreader, xmlwriter, mbstring, openssl, curl, enchant
, exif, fileinfo, ftp, gd, gettext, imagick, mysqli, odbc, Phar, pdo_mysql, pdo_pgsql, pdo_sqlite, soap, sockets, sodium, sqlite3, timezonedb, xsl

### LARAVEL VERSION:
10.48.12.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.3.4
backpack/crud: 6.7.15
backpack/theme-tabler: 1.2.10

**Reply by jcastroa87** [reply]
Is look like tabler is installed, in any case, did you try the manual way?

```
composer require backpack/theme-tabler
```

Cheers.

**Reply by DonchenkoIgor** [reply]
yes, I tried to install it manually, but still the error remained

**Reply by DonchenkoIgor** [reply]
PS D:\main-project\MyProject> php artisan backpack:install                   

   STEP 1  Installing Backpack CRUD.

  Publishing configs, views, routes and assets ........................................................................... DONE
  Creating users table ................................................................................................... DONE
  Creating CheckIfAdmin middleware ....................................................................................... DONE
  Installing Generators .................................................................................................. DONE
  Installing Basset ...................................................................................................... DONE

   STEP 2  Installing a Theme.

  │ Choose your admin UI, depending on your project and preferences.

  1 Backpack Pro ................................................................................................ NOT INSTALLED
    Adds 5 operations, 10 filters, 28 fields, 6 columns, charts

  2 DevTools .................................................................................................... NOT INSTALLED
    Helps generate models, migrations, operations and CRUDs

  3 Editable Columns ............................................................................................ NOT INSTALLED
    Allow your admins to make small changes from the table view


   INFO  Checking Backpack Basset installation.

  Initializing basset check ................................................................................................................... DONE
  Checking cache storage ...................................................................................................................... DONE
  Fetching a basset ........................................................................................................................... DONE


   DONE  Backpack installation complete.

  │ Go to http://127.0.0.1:8000/admin/ to access your new admin panel.
  │ You may need to run php artisan serve to serve your Laravel project.

**Reply by DonchenkoIgor** [reply]
URL: http://127.0.0.1:8000/admin/login

I get a 500 error when clicking on this link
 http://127.0.0.1:8000/admin/

**Reply by DonchenkoIgor** [reply]
the error looks like this:
ERROR
500
It's not you, it's me.

**Comment by jcastroa87** [comment]
Hello @DonchenkoIgor 

What is the actual domain you have on your server? and how did you config `APP_URL`in .env?

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
